stable/dex/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: dex
-version: 2.3.2
+version: 2.3.3
```

```suggestion
version: 2.4.0
```

This introduce a new feature, SemVer wants us to bump MINOR version.